internal/repository/favorites.go

```diff
@@ -0,0 +1,23 @@
+package repository
+
+import (
+	//"fmt"
+
+	"gwi_api/internal/domain"
+)
+
+type FavoritesQuery interface {
+	AddFavorites(userId uint64, assetId uint64) error
+	GetFavorites(userId uint64, params domain.PaginationParams) ([]domain.Asset, error)
+}
+
+type favoritesQuery struct{}
+
+func (f *favoritesQuery) AddFavorites(userId uint64, assetId uint64) error {
+	return DB.AddFavorites(userId, assetId)
```

Comment:
How could you mock the DB implementation in order, for example, write some unit tests for the repository layer?